config/settings/test.yml

```diff
@@ -402,3 +402,10 @@ ask_va:
   authentication:
     max_auth_retry_limit: 3
     retry_attempt_expiry: 604800 # 7 Days
+
+vye:
+  # NOTE: This is a fake key for testing purposes only.
+  ivr_key: 'testing-vye-ivr-key'
+  scrypt:
+    # NOTE: This is a random salt for testing purposes only.
+    salt: 'testing-vye-scrypt-salt'
```

Comment:
Is there a specific reason for these values to be set for testing purposes? Can you try to remove these additions and see if your tests run properly? You have added them to config/setting.yml, so you might not need these in this file.